ledger/acctupdates.go

```diff
@@ -930,10 +930,10 @@ func (au *accountUpdates) roundOffset(rnd basics.Round) (offset uint64, err erro
 	return off, nil
 }
 
-func (au *accountUpdates) commitSyncer() {
+func (au *accountUpdates) commitSyncer(deferedCommits chan deferedCommit) {
```

Comment:
passing the `deferedCommits` over the stack instead of the struct allows us to replace the one on the struct.